chingu.gemspec

```diff
@@ -1,25 +1,29 @@
-# -*- encoding: utf-8 -*-
-require File.dirname(__FILE__) + '/lib/chingu/version'
+
+require_relative 'lib/chingu/version'
 
 Gem::Specification.new do |s|
-  s.name = "chingu"
+  s.name = 'chingu'
```

Comment:
This is moving into the opposite direction of what happened to Gosu, where I've used [`rufo`](https://github.com/ruby-formatter/rufo) to turn all the single quotes into double quotes consistently. I guess that clashes with the RuboCop default settings? It would definitely avoid a lot of diff in this repository and keep things in the Gosu org more consistent to use "" (almost) everywhere.

Reply:
I see, I did the changes basing myself on RuboCop default config as you said, but I agree that a lot of differences would look bad so I'll revert to double quotes

Reply:
Thanks :)